tests/check_framework/test_urls.py

```diff
@@ -4,12 +4,12 @@
 
 
 class CheckUrlsTest(SimpleTestCase):
-    @override_settings(ROOT_URLCONF='check_framework.urls_no_warnings')
+    @override_settings(ROOT_URLCONF='check_framework.urlconfs.no_warnings')
     def test_include_no_warnings(self):
         result = check_url_config(None)
         self.assertEqual(result, [])
 
-    @override_settings(ROOT_URLCONF='check_framework.urls_include')
+    @override_settings(ROOT_URLCONF='check_framework.urlconfs.include')
```

Comment:
I was mistaken about the clash -- I was thinking the tests were in a module urls.py. They are actually in test_urls.py so there is no clash.

I have given the url files more specific names, that match the test methods.

The actual views don't matter, so I have used lambdas as suggested.